docs/_articles/en/soql/soql-builder.md

```diff
@@ -27,30 +27,38 @@ Next, install the SOQL Builder extension from [Visual Studio Marketplace](https:
 
 ## Launch SOQL Builder
 
-Launch SOQL Builder from within a Salesforce DX project.
+Launch SOQL Builder from within a Salesforce DX project. How you launch SOQL Builder depends on if you have an existing `.soql` file or if you plan to create one.
 
 ### Prerequisites
 
 - In VS Code, open a Salesforce DX project.
 - Authorize the org whose objects you want to query.
 
+### Open an Existing SOQL File in SOQL Builder
+
 DX projects have a sample `accounts.soql` file in the `<project-folder>/scripts/soql` directory. However, you can create and store your `.soql` files in any directory.
 
 1. (If necessary) Create a `.soql` file.
 1. Right-click the file name, select **Open With**, then **SOQL Builder**.
 
 ![Right-click to open SOQL Builder](./images/soql-builder-open.gif)
 
+### Launch SOQL Builder and Create a Query
+
+1. From the command palette, run **SFDX: Create Query in SOQL Builder**.
+1. Click **File > Save** to save the query. Make sure to retain the `.soql` file extension.
+
 ## Build a Query
 
 As you build your query, watch SOQL Builder display the query syntax while it simultaneously updates the `.soql` file. After you’re done building your statements, click **Run Query** to view the output.
 
 ![Build your query](./images/soql-builder.gif)
 
+**Tip:** If using the text editor to build your query, you can validate your syntax by turning on the SOQL Editor Remote Checks setting.
+
 **Beta Limitations:**
 
 - SOQL Builder currently supports interactively building simple queries. We plan to add more functionality soon. However, you can still open a more complex `.soql` file and run the query from within SOQL Builder, but you must use a text editor to update it.
-- The Run Query button gives no indication of status while it’s building the results. It can take several minutes to build a large result set. Try to resist the urge to click Run Query again.
 - When selecting fields, you can select (click) only one at a time.
 - Every time you click Run Query, a SOQL Query Results tab appears. There’s no way to associate the results with the specific query statements. The SOQL Builder editor reflects your most-recent updates.
 - If you manually update the `.soql` file using a text editor, we don’t currently support any of the syntax validation that comes with the Apex Language Server.
```

Comment:
@dobladez @dehru - is the last bullet still true given you can turn on the validator - don't know if that's the same as Apex LS, however. Please confirm if this bullet should stay or go.

Reply:
Fernando will know

Reply:
@dobladez 
@dehru - I guess we can leave that last bullet point for now - but would like to know if and when I should remove it.

Reply:
Sorry I missed your comment. You can remove it. The syntax check has been there for a while now, and it's always on. The new option is for additional server-side validation, which can catch more errors.

Reply:
@dobladez - thank you. @dehru - I'm going to approve so I don't hold up the ship. I will make a note to remove the last bullet in a future release.